packages/react-component-library/src/components/TopLevelNavigation/SearchBar/partials/StyledSearchBar.tsx

```diff
@@ -9,6 +9,6 @@ export const StyledSearchBar = styled.div<StyledSearchBarProps>`
   z-index: ${zIndex('dropdown', 1)};
   display: block;
   margin-top: -1px;
-  position: absolute;
+  position: relative;
```

Comment:
I think the fix for this possibly a z-index change:

`z-index: ${zIndex('masthead', 1)};`

Lets have a call to check.